src/Elasticsearch/ConnectionPool/Selectors/RoundRobinSelector.php

```diff
@@ -29,8 +29,10 @@ class RoundRobinSelector implements SelectorInterface
      */
     public function select($connections)
     {
+        $returnConnection = $connections[$this->current % count($connections)];
```

Comment:
Doesn't particularly matter since the connections are shuffled at the start, but can't hurt to fix while we're visiting the code :)